index.md

```diff
@@ -2,13 +2,15 @@
 
 ## Upcoming events
 
-We are having a social meetup at BRLO in Gleisdreieck this **Tuesday, August 13th at 19:00**.
+We are having meetup at [WIAS](https://www.wias-berlin.de/) this **Tuesday, October 15th at 19:00**. 
+Attention, this time it will be on the **4th floor**.
+(Mohrenstr. 39, 10117 Berlin)
 
-Please follow messages on the [ `#berlinusergroup`](https://julialang.zulipchat.com/#narrow/stream/249782-berlinusergroup) channel of the [Julia Zulip](https://julialang.zulipchat.com/) for updates and the exact location.
+This time, the discussion might focus on Julia v1.11 and related stuff.
 
 ## Regular meeting
 
-**Date/Time :** We usually meet every 2nd Tuesday of the month at 19:00.
```

Comment:
While we could do a better job at sticking to it, I'd still say the 2nd is the expected "regular meeting" time.